tools/example-templates/additional-files/SimpleAttribute.cs

```diff
@@ -0,0 +1,4 @@
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface)]
+public class SimpleAttribute : Attribute
```

Comment:
Again this is effectively the same as Attr1Attribute. We should consider whether we can simplify the standard.